pyrevitlib/pyrevit/revit/db/query.py

```diff
@@ -524,7 +525,7 @@ def get_linked_model_doc(linked_model):
 
     if lmodel:
         for open_doc in HOST_APP.docs:
-            if open_doc.Title == lmodel.name:
+            if os.path.split(open_doc.PathName)[1] == lmodel.name:
```

Comment:
open_doc.Title did return filename without extension, and lmodel.name does with extenstion